### PR TITLE
add our networks button

### PR DESCRIPTION
**Pull request by ayumitk**
Screenshot 2024-03-06 at 7 09 07 AM

- Added our networks button

### CODE REVIEW

**Review by impelcrypto — APPROVED** [review]

**Comment by gtg7784** [comment]
@ayumitk I think this PR is outdated, could you please confirm to close this PR?

File: lang/ko-home.json [hunk 1]
```diff
@@ -98,7 +98,10 @@
           "developerExperience": "이더리움의 검증된 스택, 툴링 및 인프라에 의한 친숙한 개발자 경험"
         }
       },
-      "cta": "개발 시작"
+      "cta": {
+        "startBuilding": "개발 시작",
```
Comment (gtg7784) [inline lang/ko-home.json:102]: Please update as 시작하기

File: lang/ko-home.json [hunk 2]
```diff
@@ -98,7 +98,10 @@
...
+        "ourNetworks": "우리 네트워크"
```
Comment (gtg7784) [inline lang/ko-home.json:103]: also please update as 네트워크 알아보기

**Review by gtg7784 — CHANGES_REQUESTED** [review]
Please update those KR texts, thank you :)

File: lang/ko-home.json (hunk 1 above)
Comment (ayumitk) [reply lang/ko-home.json]: @gtg7784 Updated in [5038e3b](https://github.com/AstarNetwork/astarwebsite_v2/pull/347/commits/5038e3bd92f03919905fc42baa12bea91018e5e8)

File: lang/ko-home.json (hunk 2 above)
Comment (ayumitk) [reply lang/ko-home.json]: @gtg7784 Updated in [5038e3b](https://github.com/AstarNetwork/astarwebsite_v2/pull/347/commits/5038e3bd92f03919905fc42baa12bea91018e5e8)

**Review by gtg7784 — APPROVED** [review]
LGTM!

**Pull request merged**